examples/whisper/requirements.txt

```diff
@@ -1,7 +1,7 @@
 neural-compressor
 onnx==1.14.0
-onnxruntime>=1.15.0
-onnxruntime-extensions>=0.8.0
+onnxruntime==1.15.0
```

Comment:
Is this version fix for onnxruntime because of the bug in quantization preprocess?

Reply:
I just checked locally and the whisper example works fine with ort 1.16.0. This version also has multilingual support so nightly version is not required for that anymore.

Reply:
reverted